relay-server/src/envelope.rs

```diff
@@ -1003,6 +1013,11 @@ impl Item {
             ItemType::Unknown(_) => false,
         }
     }
+
+    pub fn into_parts(self) -> (ItemHeaders, Bytes) {
```

Comment:
Missing docs, isn't there a clippy lint 🤔

Reply:
Removed in favor of destructuring.